src/CalculatorUnitTests/UnitConverterViewModelUnitTests.cpp

```diff
@@ -827,7 +851,9 @@ namespace CalculatorUnitTests
             WaitForSingleObjectEx(GetCurrentThread(), 1100, FALSE);
             VERIFY_ARE_EQUAL((UINT)3, vm.SupplementaryResults->Size);   // Verify we're in the state we expect as a pre condition
 
-            vm.Unit1 = vm.Units->GetAt(2);
+            auto ds = vm.SupplementaryResults->Size;
```

Comment:
Seems like left over debugging code? Or I might be missing something here.

Reply:
debug trace yes. Removed